source/PlayState.hx

```diff
@@ -174,6 +176,29 @@ class PlayState extends MusicBeatState
 				dialogue = CoolUtil.coolTextFile(Paths.txt('thorns/thornsDialogue'));
 		}
 
+		#if !html
+		// Making difficulty text for Discord Rich Presence.
+		var storyDifficultyText = "";
+		switch (storyDifficulty)
+		{
+			case 0:
+				storyDifficultyText = "Easy";
+			case 1:
+				storyDifficultyText = "Normal";
+			case 2:
+				storyDifficultyText = "Hard";
+		}
```

Comment:
This might be able to be done easier with one of the additions to CoolUtil.hx in commit 3d46c
Basically use PlayState.storyDifficulty and toss it into the CoolUtil function or whatev. Just a little reminder for myself later!!